### ISSUE TITLE
freebayes parallel missing options

### ISSUE THREAD

**Issue by davidvilanova** [freebayes/freebayes#519]
Hi,
I´m running the following command:

`freebayes -f genome.fna --populations  population.txt -L bamfiles.txt -p 1 --min-coverage 10  --min-coverage 20 > freebayes.vcf
`
Is there a quick way to run this command in parallel ?? the freebayes-parallel does not have the population option.

**Comment by ekg** [comment]
freebayes-parallel?

**Comment by davidvilanova** [comment]
I cannot run the same command with freebayes-parallel because the 

> "--populations"

 option and the

>  "-L"

 are not available in the parallel version. It seems the parallel version is restricted to one bam file without all other options.

**Comment by dfajar2** [comment]
I found an alternate way to run multiple jobs at a time that may help without using freebayes-parallel . 
This is what I did. Suppose you have a bed file with 500 target regions.   
1. Split the regions into separate files with ~50 subregions each.  
2. Created a bash file for each subregion (100 in total)
3. Installed a hidden gem I found called Task Spooler (http://vicerveza.homeunix.net/~viric/soft/ts/)
4. Set up ts to run a maximum of i.e. 30 jobs simultaneously 
5. Submit all the jobs using ts and that's it.
